YvesIraguha/TeaHouse
# #170533982 Fix Submission Message

#### What does this PR do?
- Fix paginations and sort all pieces from latest
#### Description of Task to be completed?
- include the type of submitted output in the email to a user
- Replace images type with Images
#### How should this be manually tested?
- Clone the repo and checkout the branch `bg-fix-submission-message-170533982`
- Run `npm i && npm run precommit && npm run dev:start` to run tests and start the server
- Submit a piece, you should see the outcome. 

#### What are the relevant pivotal tracker stories?
#[170533982](https://www.pivotaltracker.com/story/show/170533982)
